src/pages/en/guides/migrate-to-astro/from-gatsby.mdx

```diff
@@ -2,67 +2,576 @@
+Use `npm create astro@latest` to launch Astro's CLI wizard or choose a community theme from the [Astro Theme Showcase](https://astro.build/themes). 
 
-You can pass a `--template` argument to the `create-astro` command to start a new Astro project from any existing Astro repository on GitHub:
+
+You can also pass a `--template` argument to the `create-astro` command to start from an existing Astro repository on GitHub:
 
   <PackageManagerTabs>
     <Fragment slot="npm">
     ```shell
-    npm create astro -- --template withastro/blog
+    npm create astro -- --template <github-username>/<github-repo>
     ```
     </Fragment>
     <Fragment slot="pnpm">
     ```shell
-    pnpm create astro --template withastro/blog
+    pnpm create astro --template <github-username>/<github-repo>
     ```
     </Fragment>
     <Fragment slot="yarn">
     ```shell
-    yarn create astro --template withastro/blog
+    yarn create astro --template <github-username>/<github-repo>
     ```
     </Fragment>
   </PackageManagerTabs>
 
-Bring your existing Markdown (or MDX, with our optional integration) files as content to [create Markdown or MDX pages](/en/guides/markdown-content/).
+Then, copy your existing Gatsby project files over to your new Astro project into a separate folder outside of `src`.
+
+:::tip
+Prefer to explore in your browser instead? Visit https://astro.new for a variety of starter templates that can be opened and edited in StackBlitz, CodeSandbox or Gitpod.
+:::
+
+### Install integrations (optional)
+
+You may find it useful to install some of [Astro's optional integrations](/en/guides/integrations-guide/) to use while converting your Gatsby project to Astro:
+
+- **@astrojs/react**: to reuse some existing React UI components in your new Astro site, or keep writing with React components.
+
+- **@astrojs/image**: to replace Gatsby's Image plugin with Astro's own image-optimizing components. (Experimental: works in `.astro` and `.mdx` files only.)
+
+- **@astrojs/mdx**: to bring existing MDX files from your Gatsby project, or to use MDX in your new Astro site.
+
+### Repurpose config files
+
+Gatsby has several top-level configuration files that also include site and page metadata and are used for routing. You will not use any of these `gatsby-*.js` files in your Astro project, but there may be some content that you can reuse:
+
+- `gatsby-config.js`: Move your `siteMetadata: {}` into `src/data/siteMetadata.js` (or `siteMetadata.json`) to import data about your site (title, description, social accounts etc.) into page layouts.
+
+- `gatsby-browser.js`: Consider adding anything used here directly into your main layout's `<head>` tag.
+
+- `gatsby-node.js`: You will not need to create your own nodes in Astro, but viewing the schema in this file may help you with defining types in your Astro project.
+
+- `gatsby-ssr.js`: If you choose to use SSR in Astro, you will add and configure the adapter of your choice directly in `astro.config.mjs`.
```

Comment:
Reminder to take a pass for link opportunities? Here I noticed SSR could definitely map to the SSR docs? Probably more opportunities like that here and elsewhere.